packages/flow/lib/shared.js

```diff
@@ -1,30 +1,34 @@
 /** @type {import('./shared.d').isIterable} */
-export const isIterable = (value) => (
+export const isIterable = (value) =>
 	value != null &&
-	(
-		typeof value[Symbol.iterator] === 'function' ||
-		typeof value[Symbol.asyncIterator] === 'function'
-	)
-)
+	(typeof value[Symbol.iterator] === 'function' ||
+		typeof value[Symbol.asyncIterator] === 'function')
 
 /** @type {GeneratorFunctionConstructor} */
-const GeneratorFunction = (function * () {}).constructor
+const GeneratorFunction = function* () {}.constructor
 
 /** @type {AsyncGeneratorFunctionConstructor} */
-const AsyncGeneratorFunction = (async function * () {}).constructor
+const AsyncGeneratorFunction = async function* () {}.constructor
+
+/** @type {import('./shared.d').isGenerator} */
+const isGenerator = (value) =>
```

Comment:
I was planning on using this function but eventually realized that I didn't need it for the `iterate()` and the `<For/>` component.